composer/models/resnet/model.py

```diff
@@ -37,7 +37,7 @@ def composer_resnet(model_name: str,
         model_name (str): Name of the ResNet model instance. Either [``"resnet18"``, ``"resnet34"``, ``"resnet50"``, ``"resnet101"``,
             ``"resnet152"``].
         num_classes (int, optional): The number of classes. Needed for classification tasks. Default: ``1000``.
-        pretrained (bool, optional): If True, use ImageNet pretrained weights. Default: ``False``.
+        weights (str, optional): If provided, pretrained weights can be specified, such as with ``IMAGENET1K_V2``. Default: ``None``.
```

Comment:
any way we can avoid changing the API here?

Reply:
Hm... we could leave it as if `pretrained=True`, we take `IMAGENET1K_V2` weights (the best ones). I'm pro changing though -- imo we should mirror the API to torchvision

Reply:
So the issue is `weights` is only added in 0.13. I'm not sure if we've figured out an upgrade policy, but I see a few ways to proceed
(1) update `setup.py` to use the most recent `torchvision` release
(2) update `setup.py` to require `torchvision` that's at max 0.12
(3) update our class to identify the `torchvision` version being used and appropriately pass in the right args
I'm personally in favor of 1 > 3 > 2

Reply:
Agreed; let's not force versions.